.eslintrc

```diff
@@ -1,35 +1,33 @@
 {
-    "env": {
```

Comment:
Please, don't mess up file which you don't use

Reply:
I won't